### PR TITLE
DX-1122 MDCB 2.4.3 Release notes update

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

[MDCB-2.4](https://deploy-preview-4166--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-mdcb/release-notes/version-2.4)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

___

## **Type**
enhancement, documentation

___

## **Description**
- Added 2.4.3 Release Notes to the MDCB documentation.
- Highlighted the resolution of an issue causing partial outages in Tyk Cloud Hybrid gateways due to a blocked stats channel.
- Provided upgrade instructions and download links for the 2.4.3 release.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-2.4.md</strong><dd><code>MDCB 2.4.3 Release Notes Update</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.4.md
<li>Added 2.4.3 Release Notes section.<br> <li> Detailed the release date, breaking changes, deprecations, upgrade <br>instructions, release highlights, and download links.<br> <li> Included a changelog with a fix for a blockage in the stats channel <br>causing partial outages in Tyk Cloud Hybrid gateways.

</details>
    

  </td>
  <td><a href="https:/TykTechnologies/tyk-docs/pull/4166/files#diff-864186a874168f7c49c27f1dfcc8ca739ae09ca6299354b8f01b0e7089d31e71">+34/-0</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Review by sredxny — APPROVED** [review]
lgtm

File: tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.4.md [hunk 1]
```diff
@@ -13,6 +13,40 @@ Licensed Protected Product
...
+There are no deprecations in this release.
+
+#### Upgrade instructions
+If you are using a 2.4.x version, we advise you to upgrade ASAP to this latest release. If you are on an older version, you should skip 2.4.0 and upgrade directly to this release.
+
+#### Release Highlights
+This release resolved an issue causing partial outages in Tyk Cloud Hybrid gateways due to a blocked stats channel, affecting login RPC calls and gateway operations.
+
+#### Downloads
+- [Docker image to pull](https://hub.docker.com/layers/tykio/tyk-mdcb-docker/v2.4.3/images/sha256-xxxxxx) TBD
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.4.md]: Remember to add docker image link for this release

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi @caroltyk Carol, PR LGTM. Left a comment to update download docker image link. Are there any deprecations, and what are the [dependencies](https://tyktech.atlassian.net/wiki/spaces/EN/pages/2131918959/Guidelines+For+Writing+Tyk+Release+Notes#Dependencies) (3rd party dependencies and compatibility) for MDCB?

File: tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.4.md (hunk 1 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.4.md]:
```suggestion
- [Docker image to pull](https://hub.docker.com/layers/tykio/tyk-mdcb-docker/v2.4.3/images/sha256-832f461782fbc6182382798a89025b0489f529427521f92683f33df1ebbd4218?context=explore)
```

**Comment by carolwkc** [comment]
@dcs3spp information updated. It is ready for release once approved.

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**